server/src/database/commands/data-seed-demo-workspace.command.ts

```diff
@@ -0,0 +1,91 @@
+import { Command, CommandRunner } from 'nest-commander';
+import { DataSource } from 'typeorm';
+
+import { DataSourceService } from 'src/metadata/data-source/data-source.service';
+import { WorkspaceMigrationService } from 'src/metadata/workspace-migration/workspace-migration.service';
+import { WorkspaceMigrationRunnerService } from 'src/workspace/workspace-migration-runner/workspace-migration-runner.service';
+import { TypeORMService } from 'src/database/typeorm/typeorm.service';
+import { seedMetadataSchema } from 'src/database/typeorm-seeds/metadata';
+import { seedCoreSchema } from 'src/database/typeorm-seeds/core';
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+
+// TODO: implement dry-run
+@Command({
+  name: 'workspace:demo',
+  description:
+    'Seed workspace with initial data. This command is intended for development only.',
+})
+export class DataSeedDemoWorkspaceCommand extends CommandRunner {
+  // TODO: get workspaceId from .env
+  workspaceId = '20202020-1c25-4d02-bf25-6aeccf7e1337';
+  dataSourceId = '20202020-7f63-47a9-b1b3-6c7290ca1337';
+  workspaceSchemaName = 'workspace_1wgvd1injqtife6y4rvfb1337';
+
+  constructor(
+    private readonly environmentService: EnvironmentService,
+    private readonly dataSourceService: DataSourceService,
+    private readonly typeORMService: TypeORMService,
+    private readonly workspaceMigrationService: WorkspaceMigrationService,
+    private readonly workspaceMigrationRunnerService: WorkspaceMigrationRunnerService,
+  ) {
+    super();
+  }
+
+  async run(): Promise<void> {
+    try {
+      const dataSource = new DataSource({
+        url: this.environmentService.getPGDatabaseUrl(),
+        type: 'postgres',
+        logging: true,
+        schema: 'public',
+      });
+      await dataSource.initialize();
+
+      await seedCoreSchema(dataSource, this.workspaceId);
+      await seedMetadataSchema(
```

Comment:
I think this won't actually work because these metadata seeds have hardcoded ids.
Could you leverage:
```
    const createdObjectMetadata =
      await this.createStandardObjectsAndFieldsMetadata(
        dataSourceMetadata.id,
        workspaceId,
      );
```
instead see from workspace-manager.service.ts

Reply:
Now that I think about it, what we want to do is exactly what we have in the init method from workspace-manager.service.ts.

Change in strategy: 
1) Keep what you have to create the core entities
2) revert the changes to the typeorm-seeds/metadata
3) create a initDemo in workspace-manager.service.ts that is doing exactly what we want but instead of "prefillWorkspaceWithStandardObjects" we want "prefillWorkspaceWithDemoObjects"

Reply:
This should work like a charm, sorry I did not think about it earlier

Reply:
Thank you for your quick review, I will adjust accordingly!